Ventilators/R4/resources/Questionnaire-R4-VentilatorsProgressNote.json

```diff
@@ -87,7 +74,7 @@
               "url": "http://hl7.org/fhir/StructureDefinition/cqf-expression",
               "valueExpression": {
                 "language": "text/cql",
-                "expression": "\"VentilatorsPrepopulation\".AnswerYes"
+                "expression": "\"BasicClinicalInfoPrepopulation\".AnswerYes"
```

Comment:
Another location where `item.initial` should be used instead.

Reply:
fixed